### DISCUSSION TITLE
How can i exclude routes

### DISCUSSION THREAD

**Discussion by rabol** [dedoc/scramble#209] [Q&A] [answered]
Hi

I would like to exclude a few routes that is in my api.php routes file, how can that be done ?

**Comment by romalytvynenko** [comment, marked as answer]
Hey @rabol 

You can use `Scramble::routes(...)` in some service provider: https://scramble.dedoc.co/usage/getting-started#routes-resolution

Static `routes` method accept a function that allows you to filter which routes should be documented by Scramble. This is how you can exclude some routes.

🙌
